Synthetic Data Generation/README.md

```diff
@@ -21,18 +21,52 @@ Tested in Viya 4, Stable 2022.09
 
 ## User Interface
 
-Refer the "About" tab on each of the individual steps for more details on what they are used for.
+**Refer the "About" tab on each of the individual steps for more details on what they are used for.**
 
 ### Train a Synthetic Data Generator
 This step helps you train a synthetic data generator model, using a Generative Adversarial Network. Parameters required :
 1. Variables you wish to generate a Synthetic Data Generator for
 2. Number of sample observations, for assessment purposes
-3. Optional - training parameters such as number of epochs etc. (Advanced)
+3. Optional - additional training parameters such as number of epochs, the minibatch size, and whether to use Graphical Processing Units (GPUs). (Advanced tab)
 4. Input port - attach a training dataset
 5. Output ports - attach tables for the sample observations and the desired model binary (astore).
 
 ![Train a Synthetic Data Generator](./img/train-a-synthetic-data-generator.png)
 
+#### Run-time control
+
+**Note that this is optional.**  
```

Comment:
Add > at beginning of line to emphasize this note and visually separate it more clearly from the section header. 
So this line would read like this:

> **Note that this is optional.**

Reply:
Done